lib/rules/length-zero-no-unit/index.js

```diff
@@ -28,7 +28,11 @@ const rule = function(actual, secondary, context) {
     }
 
     root.walkDecls(decl => {
-      check(blurComments(decl.toString()), decl);
+      const stringValue = blurComments(decl.toString());
+
+      if (!/calc\((.|\s)*\)/i.test(stringValue)) {
```

Comment:
Let's move this to util and name file `isMathFunction` and put logic in utils (in next css spec we will have more math functions, not only `calc`)

Reply:
done